packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -331,15 +331,15 @@ Function ConfigAll() {
     )
 
     if ($OtelConfigLocation) {
-        Write-Output "****** processing cwagent-otel-collector ******"
+        Write-Output "****** Processing cwagent-otel-collector ******"

```

Comment:
Why are we changing this? Does this mirror what the linux ctl script already does?